game-runner/main.cpp

```diff
@@ -1,27 +1,25 @@
-#include <led-matrix.h>
+#include "led-matrix.h"
 
-#include <unistd.h>
 #include <math.h>
-#include <stdio.h>
 #include <signal.h>
```

Comment:
Looks perfect apart from this one part - could you make sure these graphics/ imports aren't being removed?